pkg/querier/block_streaming.go

```diff
@@ -37,6 +38,10 @@ type blockStreamingQuerierSeriesSet struct {
 	nextSeriesIndex int
 
 	currSeries storage.Series
+
+	// Debug logging for continuous test queries.
```

Comment:
This comment still refers to continuous-test.

(same applies to line 103 below)